index.html

```diff
@@ -6,13 +6,13 @@
   <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
   <meta http-equiv="Expires" content="0" />
   <title></title>
-  <script src="https://unpkg.com/@microsoft/office-js@custom-functions-preview-refresh/dist/custom-functions-runtime.js" type="text/javascript"></script>
-  <script src="dist/win32/ship/index.win32.bundle" type="text/javascript"></script>
+  <script src="https://officedev.github.io/custom-functions/lib/custom-functions-runtime.js" type="text/javascript"></script>
+  <script src="dist/win32/ship/index.win32.bundle.js" type="text/javascript"></script>
```

Comment:
We need to use the web vs. native bundles, like I have in my reworked templates.  It makes no sense to have "win32" in here, esp. since this file only works on the WAC!